compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/pointer/PointerEvent.skiko.kt

```diff
@@ -86,9 +91,26 @@ fun PointerKeyboardModifiers(
     return PointerKeyboardModifiers(res)
 }
 
+fun PointerKeyboardModifiers(keyEvent: KeyEvent): PointerKeyboardModifiers {
```

Comment:
It's needed here - https://github.com/JetBrains/androidx/blob/986cb733f8a4809a2c207af5c27db08cf56c236e/compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt#L297

I think we don't need to have it public. Moving it into `ComposeLayer.desktop.kt` for now as private that takes awt KeyEvent as a parameter. This would require to implement similar private function for other platforms. Or we will make it internal expect/actual.

Reply:
done